webservices/rest.py

```diff
@@ -114,6 +115,7 @@ def get(self):
         findme = ' & '.join(args['q'].split())
         data = db_conn().execute(qry, findme=findme).fetchall()
 
+        ### The count is wrong if more than 20 results are returned.
```

Comment:
Can we still use the Pagination class, but just pass in 20? No need to say "the count is wrong." It won't be unless someone purposely changes the SQL query. :)